integrationTest/integration_test.go

```diff
@@ -1263,12 +1263,6 @@ func makeTests() []*TestGroup {
 			// Vultr syntax-checks TXT records with SPF contents.
 			tc("Create a TXT/SPF", txt("foo", "v=spf1 ip4:99.99.99.99 -all")),
 
-			// Nobody needs this and many APIs don't allow it.
```

Comment:
Why remove this?  I think the comment "Nobody needs this" was sarcasm.  If an API doesn't allow it, the auditrecords.go file should indicate if this needs to be skipped.